### ISSUE TITLE
Triton not installed

### ISSUE THREAD

**Issue by eltictacdicta** [PromtEngineer/localGPT-Vision#11]
when I start the application I get this error (my graphics card is the GTX 4060):
(localgpt-vision) c:\driverslocalGPT-Vision>python app.py
WARNING 10-14 20:17:15 _custom_ops.py:18] Failed to import from vllm._C with ModuleNotFoundError(‘No module named “vllm._C”’)
INFO 10-14 20:17:16 importing.py:10] Triton not installed; certain GPU-related functions will not be available.

**Comment by PromtEngineer** [comment]
I think its just a warning. Are you able to run it after the warning?

**Comment by eltictacdicta** [comment]
Yes, it works but I can't get it to use the GPU for indexing for example, and it only works with GP4o, it doesn't work with any local model.
